zookeeper-server/src/main/java/org/apache/zookeeper/server/ZooKeeperServer.java

```diff
@@ -98,6 +98,14 @@
         Environment.logEnv("Server environment:", LOG);
     }
 
+    private static final String SHUTDOWN_TIMEOUT = "zookeeper.shutdownTimeout";
+    private static int shutdownTimeout = 10000;
+
+    static {
+        shutdownTimeout = Integer.getInteger(SHUTDOWN_TIMEOUT, 10000);
+        LOG.info(SHUTDOWN_TIMEOUT + "=" + shutdownTimeout);
+    }
+
```

Comment:
it would be good to refactor this code a little bit, by moving all static constants and property definition to a single place and consolidate all static initializer block into a single block (instead of 3).

Reply:
this has been moved to RequestThrottler